src/main/java/io/vertx/circuitbreaker/impl/HystrixMetricEventStream.java

```diff
@@ -0,0 +1,106 @@
+package io.vertx.circuitbreaker.impl;
+
+import io.vertx.circuitbreaker.CircuitBreakerState;
+import io.vertx.circuitbreaker.HystrixMetricHandler;
+import io.vertx.core.Vertx;
+import io.vertx.core.http.HttpServerResponse;
+import io.vertx.core.json.JsonObject;
+import io.vertx.ext.web.RoutingContext;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+/**
+ * Implements a handler to servers the Vert.x circuit breaker metrics as a Hystrix circuit
+ * breaker.
+ *
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public class HystrixMetricEventStream implements HystrixMetricHandler {
+
+  private final List<HttpServerResponse> connections = new ArrayList<>();
```

Comment:
I'm not sure if it should be a arraylist, http responses are short lived objects so there will be many having a array backed list will require a big sequential set of ram since we keep adding stuff to it right? My guess is that it should be a linked list.